package.json

```diff
@@ -50,6 +50,7 @@
     "lodash.trimstart": "^4.5.1",
     "lodash.upperfirst": "^4.3.1",
     "pino": "^9.0.0",
+    "prettier": "^3.0.1",
```

Comment:
Question: Shouldn't be a peer dependencies?

What happen if I will use a different formatter (like biome)?
Maybe we should run prettier, if we find it as dependency.

What you think?

Reply:
Usually generated files should not be processed by the user's formatter. We state this also on top of each generated file, as a comment.
Said so IMO it's fine to keep it here.

Reply:
ok make sense. I didn't figure out that is a feature from `json-schema-to-typescript` config.

The only thing that I don't understand why `json-schema-to-typescript` does not format the code as expected, since the user pass the option.

Reply:
The TS code currently generated is already formatted by prettier, as that's what's used in json-schema-to-typescript.

Reply:
Thank for the clarification. Now I got it.